src/ecdsa.rs

```diff
@@ -12,6 +13,8 @@ use std::array::TryFromSliceError;
 pub enum Error {
     /// Error occurred due to invalid secret key
     InvalidSecretKey,
+    /// Error occurred due to invalid public key
```

Comment:
nit: add new items (like `InvalidPublicKey`) to the end. Just in case to keep ABI compatible with previous version.

Reply:
In this case we never released, so it seems cleaner to put the key errors together